pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -125,6 +133,7 @@ func (c *collector) Run(stopCh <-chan struct{}) error {
 	}
 
 	go wait.Until(func() {
+		c.collectGPUUsage()
```

Comment:
Can we split up the collection of GPU metrics? Or make the collection configurable since not each nodes mount GPU devices.